networkx/algorithms/chordal.py

```diff
@@ -283,9 +289,15 @@ def chordal_graph_treewidth(G):
     >>> nx.chordal_graph_treewidth(G)
     3
 
+    Notes
+    -----
+    See [1]_ for a definition of and further information on treewidth.
```

Comment:
How long is the definition of tree width? Can we summarize it here and *then* say "for further info, see..."?